docs/getting-started-guides/docker.md

```diff
@@ -83,9 +83,12 @@ $ kubectl config set-context test-doc --cluster=test-doc
 $ kubectl config use-context test-doc
 ```
 
-For Max OS X users instead of `localhost` you will have to use IP address of your docker machine,
-which you can find by running `docker-machine env <machinename>` (see [documentation](https://docs.docker.com/machine/reference/env/)
-for details).
+For Max OS X users instead of `localhost` you will have to use the IP address of your docker machine. You also need to forward requests from the docker-machine ip address to `localhost` on the docker VM. Instead of the first line above use
```

Comment:
Can we preface the above instructions and these instruction as being "For Linux users" and "For Mac users" so this is a little more clear? Read from top to bottom, Mac users are going to try the first thing then get alternate instructions that actually work.